README.md

```diff
@@ -0,0 +1,34 @@
+# Hyperbolic Conservation Laws
+
+[![License](https://licensebuttons.net/l/by-nc-nd/3.0/88x31.png)](https://creativecommons.org/licenses/by-nc-nd/4.0/legalcode)
+
+This is a set of tools for numerically solving different kinds of hyperbolic conservation laws with OpenCL. Currently implemented are linear transport with constant and variable coefficients, the nonlinear magnetic induction equation and ideal MHD. The code is generalised in a form that allows 
```

Comment:
A very pedantic person might want to read "(hyperbolic) conservation or balance laws".

Reply:
If we change that,  the change should also be made below.